src/components/Blog/CustomRichBlocks.tsx

```diff
@@ -22,19 +29,45 @@ interface ImageData {
   image: MediaAttributes;
 }
 
+interface WidgetRouteSettings {
+  fromChain?: string;
+  fromToken?: string;
+  toChain?: string;
+  toToken?: string;
+  fromAmount?: string;
+  allowChains?: string;
+}
+
 export const CustomRichBlocks = ({
   id,
   baseUrl,
   content,
 }: CustomRichBlocksProps) => {
-  const theme = useTheme();
   const customRichBlocks = {
     // You can use the default components to set class names...
     link: (data: any) => {
       return <Link href={data.url}>{data.children[0].props.text}</Link>;
     },
     image: (data: ImageData) =>
       baseUrl ? <Lightbox imageData={data.image} baseUrl={baseUrl} /> : null,
+    heading: ({ children, level }: any) => {
+      switch (level) {
+        case 1:
+          return <BlogH1 variant="h1">{children}</BlogH1>;
+        case 2:
+          return <BlogH2 variant="h2">{children}</BlogH2>;
+        case 3:
+          return <BlogH3 variant="h3">{children}</BlogH3>;
+        case 4:
+          return <BlogH4 variant="h4">{children}</BlogH4>;
+        case 5:
+          return <BlogH5 variant="h5">{children}</BlogH5>;
+        case 6:
+          return <BlogH6 variant="h6">{children}</BlogH6>;
+        default:
+          return <BlogH1 variant="h1">{children}</BlogH1>;
```

Comment:
```suggestion
        case 1:
        default:
          return <BlogH1 variant="h1">{children}</BlogH1>;
```

Would be better as it is doing the same thing, you can also remove the case 1 in that case

Reply:
Good point, that does the job just as well with less code